lib/Mongo/MongoCollection.php

```diff
@@ -96,6 +110,19 @@ public function __get($name)
         return $this->db->selectCollection($this->name . '.' . $name);
     }
 
+    public function __set($name, $value)
+    {
+        if ('w' == $name) {
+            $this->w = $value;
+            $this->setWriteConcern($this->w, $this->wtimeout);
+        } elseif ('wtimeout' == $name) {
+            $this->wtimeout = $value;
+            $this->setWriteConcern($this->w, $this->wtimeout);
+        } else {
+            trigger_error('trying to set invalid property', E_ERROR);
```

Comment:
I've tried to work with this but decided to screw it and leave access to undefined members out. #13 also removes this functionality for MongoId where I had it implemented using a hash.